### ISSUE TITLE
Must set launchActivity in pressAction to launch app

### ISSUE THREAD

**Issue by mars-lan** [notifee/react-native-notifee#121]
According to https://notifee.app/react-native/docs/android/interaction#press-action it seems that setting just `id` in  `pressAction` is enough to launch the default activity when the notification is tapped. It turns out that's only true for grouped notifications. For un-grouped notification, you must manually set `launchActivity` or else it simply dismisses the notification when tapped.

**Comment by mikehardy** [comment]
Hi @mars-lan !

You may be correct there, I have a simple un-grouped notification that I show in my work project, and it works correctly but I am using it as you describe with a launchActivity, like so:

```typescript

            pressAction: {
              id: 'default',
              launchActivity: 'default',
              launchActivityFlags: [AndroidLaunchActivityFlag.SINGLE_TOP],
            },
```

We'll need to see if this is a documentation issue or a code problem but may not have investigated without your mentioning it, thanks

**Comment by mars-lan** [comment]
Argh didn't know you can set `launchActivity` to `default`. That's a nice trick.

**Comment by mikehardy** [comment]
Ha - I'm glad if that helps you but I think you still point out at minimum a documentation/behavior gap, we'll see when we check it out

**Comment by helenaford** [comment]
p/r open with fix #141

**Comment by mikehardy** [comment]
The related PR has been merged and you can either reference the module via a git reference pointing to this commit hash https://github.com/notifee/react-native-notifee/commit/0727e2c6d5b8a50f39e8249d9fd9f1be598f41f9 or any build after 12.2 (current), so 12.3 or higher will have it
